backend_modules/libvirt/host/combustion

```diff
@@ -14,38 +14,49 @@ systemctl enable sshd.service
 ${ gpg_keys }
 
 # Add repositories
-zypper ar http://${ use_mirror_images ? mirror : "download.opensuse.org" }/repositories/SUSE:/CA/15.5/ ca_suse
+zypper ar http://${ user_mirror_image ? mirror : "download.opensuse.org" }/repositories/SUSE:/CA/15.5/ ca_suse
```

Comment:
No, this looks like a typo, the value in the template is `use_mirror_image`. Where does the 'r' come from?